tests/integration/test_fim/test_synchronization/test_sync_time.py

```diff
@@ -0,0 +1,162 @@
+'''
+copyright: Copyright (C) 2015-2022, Wazuh Inc.
+
+           Created by Wazuh, Inc. <info@wazuh.com>.
+
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: Check when the 'wazuh-syscheckd' daemon is performing a synchronization, a normal synchronization will end
+before the configured `interval` and `max_interval`.
+
+components:
+    - fim
+
+suite: synchronization
+
+targets:
+    - agent
+    - manager
+
+daemons:
+    - wazuh-syscheckd
+
+os_platform:
+    - linux
+
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - Debian Buster
+    - Red Hat 8
+    - Ubuntu Focal
+    - Ubuntu Bionic
+    - Windows Server 2019
+
+
+references:
+    - https://documentation.wazuh.com/current/user-manual/capabilities/file-integrity/index.html
+    - https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/syscheck.html#synchronization
+
+pytest_args:
```

Comment:
Could you add the fim_mode args?

Reply:
Fixed in commit https://github.com/wazuh/wazuh-qa/pull/3380/commits/6b66bc842d17c0bc1f7d20dc3dcc8c96bce3d18e